backend/de.metas.externalsystem/src/test/java/de/metas/externalsystem/ExternalSystemConfigRepoTest.java

```diff
@@ -720,16 +723,9 @@ void externalSystem_LeichMehl_Config_getById()
 				.type(X_ExternalSystem_Config.TYPE_LeichMehl)
 				.build();
 
-		final ExternalSystemParentConfigId externalSystemParentConfigId = ExternalSystemParentConfigId.ofRepoId(parentRecord.getExternalSystem_Config_ID());
-
 		final I_ExternalSystem_Config_LeichMehl leichMehlConfig = ExternalSystemTestUtil.createLeichMehlConfigBuilder()
 				.externalSystemParentConfigId(parentRecord.getExternalSystem_Config_ID())
 				.value("LeichMehl")
```

Comment:
please use the existing test cases and add the new (tcp etc) properties

Reply:
the new values are present on `de.metas.externalsystem.ExternalSystemTestUtil#createLeichMehlConfig` as optional and computed using `de.metas.common.util.CoalesceUtil#coalesceNotNull(T, T)` with some default values. should I change those to be mandatory and add them on each builder ?

Reply:
- in this case: yes, please to that, because imo default tcp/connection values make no sense, when we want to talk to a hardware-device
- generally: please add also the non-mandatory properties. in the repo unit tests

Reply:
@metas-ts sorry for the misunderstanding, I was referring only to unit tests and not to the actual hardware-device connection. There is a helper method to create the test data for the repo unit tests and within that helper all the available fields on `I_ExternalSystem_Config_LeichMehl` are set.